extension/docs/CancelRefundPayment.md

```diff
@@ -0,0 +1,59 @@
+**Please see [Integration Guide](IntegrationGuide.md) first before continuing with this document.**
+
+## Cancel or refund a payment
+
+1. Shop updates a payment with [a new transaction](https://docs.commercetools.com/http-api-projects-payments#add-transaction)
+    1. for refund
+        ```
+        {
+          "version": ${currentVersion},
+          "actions": [
+            {
+              "action": "addTransaction",
+              "transaction": {
+                "type": "Refund",
+                "amount": {
+                  "currencyCode": "§{currencyCode}",
+                  "centAmount": ${centAmount}
+                },
+                "state": "Initial"
+              }
+            }
+          ]
+        }
+        ```
+    1. for cancel authorization
+        ```
+        {
+          "version": ${currentVersion},
+          "actions": [
+            {
+              "action": "addTransaction",
+              "transaction": {
+                "type": "CancelAuthorization",
+                "amount": {
+                  "currencyCode": "§{currencyCode}",
+                  "centAmount": ${centAmount}
+                },
+                "state": "Initial"
+              }
+            }
+          ]
+        }
+        ```
+    Both transaction types works the same, because Adyen will always pick the right action for the current transaction.
+    If the transaction is not authorized yet, Adyen will try to cancel.
+    If the transaction is already charged, Adyen will do refund.
+1. Extension module finds the first `Payment.transactions` with a transaction `type='Charge' and state='Success'`. 
+Extension module takes `amount` and `transactionId` from this transaction and makes a Cancel or refund request.  
```

Comment:
```suggestion
Extension module takes `amount` and `transactionId` from this transaction and makes a Cancel or Refund request.  
```

Reply:
Actually the request is called `Cancel or refund`. It's not Cancel request or Refund request.